demo/setup/update_demo.go

```diff
@@ -45,24 +46,40 @@ func main() {
 	}
 
 	//copy the tags from the ec2.instance to their ec2.NetworkInterface
-	for _, instance := range input.getResources(map[string]string{"core.type": "ec2.Instance"}) {
+	for _, instance := range input.getResources(map[string]interface{}{"core.type": "ec2.Instance"}) {
```

Comment:
```suggestion
	for _, instance := range input.getResources(map[string]any{"core.type": "ec2.Instance"}) {
```

suggestion(non-blocking): In 1.18, `any` is preferred over `interface{}`.

Ditto for the rest of this file.